Better OSM explanation in login screen (#9384)
organicmaps/organicmaps: data/strings/strings.txt

    zh-Hans = 截至 %@ 的社区创建的 OpenStreetMap 数据。请访问 OpenStreetMap.org 以了解有关如何编辑和更新地图的信息。
    zh-Hant = 截至 %@ 的社群創建的 OpenStreetMap 資料。請訪問 OpenStreetMap.org 以了解有關如何編輯和更新地圖的更多資訊。

  [login_osm_presentation]
    comment = OSM explanation on Android login screen
    tags = android

>     en = OpenStreetMap.org (OSM) is a community project to build a free and open map. It's the main source of map data in Organic Maps and works similar to Wikipedia. You can add or edit places and they become available to millions of users all over the World.\nJoin the community and help to make a better map for everyone!

Reviewer: ```suggestion
    en = OpenStreetMap.org (OSM) is an open community project to build a free map. It's the main source of map data in Organic Maps and works similarly to Wikipedia. You can add or edit places and they become available to millions of users all over the World.\nJoin the community and help to make a better map for everyone!
```

Author: I didn't mean "open" in the sense that any one can join, that's covered with "community", but "open" in the sense of open data.

Reviewer 2: yeap, "free and open map" sounds better to me than just "free map" (sounds like gratis is the main property of it..)

Reviewer 3: We may also use "libre" word to avoid confusion of "open/free".

Author: I wouldn't use "libre" as too many people don't know what it means.

    af = OpenStreetMap.org (OSM) is 'n gemeenskapsprojek om 'n gratis en oop kaart te bou. Dit is die hoofbron van kaartdata in Organic Maps en werk soortgelyk aan Wikipedia. Jy kan plekke byvoeg of wysig en hulle word beskikbaar vir miljoene gebruikers regoor die wêreld.\nSluit aan by die gemeenskap en help om 'n beter kaart vir almal te maak!
    ar = OpenStreetMap.org (OSM) هو مشروع مجتمعي لبناء خريطة مجانية ومفتوحة. إنه المصدر الرئيسي لبيانات الخرائط في الخرائط العضوية ويعمل على غرار ويكيبيديا. يمكنك إضافة الأماكن أو تعديلها وتصبح متاحة لملايين المستخدمين في جميع أنحاء العالم. \انضم إلى المجتمع وساعد في إنشاء خريطة أفضل للجميع!
    az = OpenStreetMap.org (OSM) pulsuz və açıq xəritə yaratmaq üçün icma layihəsidir. O, Üzvi Xəritələrdə xəritə məlumatlarının əsas mənbəyidir və Vikipediyaya oxşar işləyir. Siz yerlər əlavə edə və ya redaktə edə bilərsiniz və onlar bütün dünyada milyonlarla istifadəçi üçün əlçatan olur.\nİcmaya qoşulun və hər kəs üçün daha yaxşı xəritə yaratmağa kömək edin!

>     be = OpenStreetMap.org (OSM) - гэта грамадскі праект па стварэнні бясплатнай і адкрытай карты. Гэта асноўная крыніца картаграфічных даных у Organic Maps і працуе падобна на Вікіпедыю. Вы можаце дадаваць або рэдагаваць месцы, і яны стануць даступнымі для мільёнаў карыстальнікаў па ўсім свеце.\nДалучайцеся да супольнасці і дапамажыце зрабіць лепшую карту для ўсіх!

Reviewer: ```suggestion
    be = OpenStreetMap.org (OSM) - гэта грамадскі праект па стварэнні бясплатнай і адкрытай мапы, і асноўная крыніца картаграфічных даных у Organic Maps, якая працуе падобна на Вікіпедыю. Кожны можа дадаваць або рэдагаваць месцы, якія пабачуць мільёны карыстальнікаў па ўсім свеце.\nДалучайцеся да супольнасці і дапамажыце зрабіць лепшую мапу для ўсіх!
```

    fa = OpenStreetMap.org (OSM) یک پروژه اجتماعی برای ساخت یک نقشه رایگان و باز است. این منبع اصلی داده های نقشه در نقشه های ارگانیک است و مشابه ویکی پدیا کار می کند. میu200cتوانید مکانu200cها را اضافه یا ویرایش کنید تا در دسترس میلیونu200cها کاربر در سراسر جهان قرار گیرند.\nبه انجمن بپیوندید و به ایجاد نقشه بهتر برای همه کمک کنید!
    fi = OpenStreetMap.org (OSM) on yhteisöprojekti, jonka tarkoituksena on rakentaa ilmainen ja avoin kartta. Se on Organic Mapsin tärkein karttatietojen lähde ja toimii samalla tavalla kuin Wikipedia. Voit lisätä tai muokata paikkoja, ja ne tulevat miljoonien käyttäjien saataville kaikkialla maailmassa.\nLiity yhteisöön ja auta tekemään parempi kartta kaikille!
    fr = OpenStreetMap.org (OSM) est un projet communautaire visant à créer une carte libre et ouverte. C'est la principale source de données cartographiques d'Organic Maps et son fonctionnement est similaire à celui de Wikipédia. Vous pouvez ajouter ou modifier des lieux et ils deviennent accessibles à des millions d'utilisateurs dans le monde entier.\nRejoignez la communauté et aidez-nous à créer une meilleure carte pour tout le monde !

>     he = OpenStreetMap.org (OSM) הוא פרויקט קהילתי לבניית מפה חופשית ופתוחה. זהו המקור העיקרי לנתוני מפות במפות אורגניות ופועל בדומה לוויקיפדיה. אתה יכול להוסיף או לערוך מקומות והם הופכים לזמינים למיליוני משתמשים בכל רחבי העולם.\nהצטרף לקהילה ועזור ליצור מפה טובה יותר לכולם!

Reviewer 4: ```suggestion
    he = OpenStreetMap.org (OSM) הוא פרויקט קהילתי לבניית מפה חופשית ופתוחה. זהו המקור העיקרי לנתוני מפות ב-Organic Maps והוא פועל בדומה לוויקיפדיה. ניתן להוסיף או לערוך מקומות והם הופכים לזמינים למיליוני משתמשים בכל רחבי העולם.\nהצטרף לקהילה ועזור ליצור מפה טובה יותר לכולם!
```

Please take a screenshot of how Hebrew looks in the app, either before or after accepting my changes. It may be necessary to force the text direction in the code to be RTL instead of LTR.

If not done in code, it can be done in the text by adding a right-to-left mark U+200F to the start of the string. I don't know if adding Unicode characters with `\uXXXX` is possible. It's an invisible character so adding it literally could be trouble (for future editing).

```suggestion
    he = \u200FOpenStreetMap.org (OSM) הוא פרויקט קהילתי לבניית מפה חופשית ופתוחה. זהו המקור העיקרי לנתוני מפות ב-Organic Maps והוא פועל בדומה לוויקיפדיה. ניתן להוסיף או לערוך מקומות והם הופכים לזמינים למיליוני משתמשים בכל רחבי העולם.\nהצטרף לקהילה ועזור ליצור מפה טובה יותר לכולם!
```

Author: @NeatNit 
This is how Hebrew looks like now (with \u200F). Is the text displayed correctly?

![Screenshot_2024-11-01-15-24-40-767_app organicmaps debug](https://github.com/user-attachments/assets/7e8e9b7f-c806-41c4-9ca1-270611b0cc5f)

Reviewer 4: Sorry for the late response. Yes, this is good.

    it = OpenStreetMap.org (OSM) è un progetto comunitario per costruire una mappa libera e aperta. È la principale fonte di dati cartografici di Organic Maps e funziona in modo simile a Wikipedia. Può aggiungere o modificare luoghi e questi diventano disponibili per milioni di utenti in tutto il mondo.\nEntri a far parte della comunità e contribuisca a creare una mappa migliore per tutti!
    ja = OpenStreetMap.org(OSM)は、フリーでオープンな地図を構築するコミュニティ・プロジェクトです。OSMは、Organic Mapsの地図データの主要なソースであり、ウィキペディアと同じように機能します。あなたが場所を追加したり編集したりすると、世界中の何百万人ものユーザーがその場所を利用できるようになります！
    ko = 오픈스트리트맵(OSM)은 무료 오픈스트리트맵을 구축하기 위한 커뮤니티 프로젝트입니다. 오가닉 맵의 지도 데이터의 주요 소스이며 위키백과와 유사하게 작동합니다. 장소를 추가하거나 편집하면 전 세계 수백만 명의 사용자가 사용할 수 있습니다.\n커뮤니티에 참여하여 모두를 위한 더 나은 지도를 만드는 데 도움을 주세요!
    lt = OpenStreetMap.org (OSM) - tai bendruomenės projektas, kuriuo siekiama sukurti nemokamą ir atvirą žemėlapį. Tai pagrindinis žemėlapių duomenų šaltinis "Organiniuose žemėlapiuose" ir veikia panašiai kaip "Vikipedija". Galite pridėti ar redaguoti vietoves, ir jos tampa prieinamos milijonams naudotojų visame pasaulyje.\nPrisijunkite prie bendruomenės ir padėkite kurti geresnį žemėlapį visiems!

>     lv = OpenStreetMap.org (OSM) ir kopienas projekts, kura mērķis ir izveidot bezmaksas un atvērtu karti. Tas ir galvenais karšu datu avots Organic Maps un darbojas līdzīgi kā Wikipedia. Jūs varat pievienot vai rediģēt vietas, un tās kļūst pieejamas miljoniem lietotāju visā pasaulē.\npievienojieties kopienai un palīdziet izveidot labāku karti visiem!

Reviewer 3: OpenStreetMap.org (OSM) ir kopienas projekts, kura mērķis ir izveidot atvērtajos datos balstītu karti, kura ir pieejama bez maksas. Tas ir Organic Maps galvenais karšu datu avots un darbojas līdzīgi kā Vikipēdija. Jūs varat pievienot vai rediģēt vietas, un tās kļūst pieejamas miljoniem lietotāju visā pasaulē.\nPievienojieties kopienai, lai palīdzētu izveidot labāku karti visiem!

    nb = OpenStreetMap.org (OSM) er et fellesskapsprosjekt for å bygge et gratis og åpent kart. Det er hovedkilden til kartdata i Organic Maps og fungerer på samme måte som Wikipedia. Du kan legge til eller redigere steder, og de blir tilgjengelige for millioner av brukere over hele verden. \nBli med i fellesskapet og bidra til å lage et bedre kart for alle!
    nl = OpenStreetMap.org (OSM) is een gemeenschapsproject om een vrije en open kaart te bouwen. Het is de belangrijkste bron van kaartgegevens in Organic Maps en werkt ongeveer hetzelfde als Wikipedia. U kunt plaatsen toevoegen of bewerken en deze worden dan beschikbaar voor miljoenen gebruikers over de hele wereld.\nSluit u aan bij de gemeenschap en help mee om een betere kaart voor iedereen te maken!
    pl = OpenStreetMap.org (OSM) to projekt społecznościowy mający na celu stworzenie darmowej i otwartej mapy. Jest to główne źródło danych mapowych w Organic Maps i działa podobnie do Wikipedii. Można dodawać lub edytować miejsca, które stają się dostępne dla milionów użytkowników na całym świecie.\nDołącz do społeczności i pomóż stworzyć lepszą mapę dla wszystkich!

>     pt = O OpenStreetMap.org (OSM) é um projeto comunitário para construir um mapa livre e aberto. É a principal fonte de dados de mapas no Organic Maps e funciona de forma semelhante à Wikipedia. Pode adicionar ou editar locais e estes ficam disponíveis para milhões de utilizadores em todo o mundo.\nJunte-se à comunidade e ajude a criar um mapa melhor para todos!
>     pt-BR = O OpenStreetMap.org (OSM) é um projeto comunitário para criar um mapa gratuito e aberto. Ele é a principal fonte de dados de mapas no Organic Maps e funciona de forma semelhante à Wikipedia. Você pode adicionar ou editar lugares e eles ficam disponíveis para milhões de usuários em todo o mundo.\nJunte-se à comunidade e ajude a criar um mapa melhor para todos!

Reviewer 5: ```suggestion
    pt = O OpenStreetMap.org (OSM) é um projeto comunitário aberto para construir um mapa gratuito. O OSM é a principal fonte de dados de mapas do Organic Maps e funciona de forma semelhante à Wikipedia. Pode adicionar ou editar locais e estes ficam disponíveis para milhões de utilizadores no mundo todo.\nJunte-se à comunidade e ajude a criar um mapa melhor para todos!
    pt-BR = O OpenStreetMap (OSM) é um projeto comunitário aberto para construir um mapa gratuito. O OSM é a principal fonte de dados de mapas do Organic Maps e funciona de forma semelhante à Wikipédia. Você pode adicionar ou editar lugares e eles ficam disponíveis para milhões de usuários no mundo todo.\nJunte-se à comunidade e ajude a criar um mapa melhor para todos!
```

    ro = OpenStreetMap.org (OSM) este un proiect comunitar pentru construirea unei hărți libere și deschise. Este principala sursă de date cartografice în Organic Maps și funcționează similar cu Wikipedia. Puteți adăuga sau edita locuri, iar acestea devin disponibile pentru milioane de utilizatori din întreaga lume.\nAlăturați-vă comunității și ajutați la crearea unei hărți mai bune pentru toată lumea!

>     ru = OpenStreetMap.org (OSM) - это проект сообщества по созданию бесплатной и открытой карты. Он является основным источником картографических данных в Organic Maps и работает подобно Википедии. Вы можете добавлять или редактировать места, и они становятся доступными для миллионов пользователей по всему миру.\nПрисоединяйтесь к сообществу и помогите сделать лучшую карту для всех!

Reviewer: ```suggestion
    ru = OpenStreetMap.org (OSM) - это открытый проект сообщества по созданию бесплатной карты. Он является основным источником картографических данных в Organic Maps и работает как Википедия. Кто угодно может добавлять или редактировать места, которые позже увидят миллионы пользователей по всему миру.\nПрисоединяйтесь к сообществу и помогите сделать лучшую карту для всех!
```

    sw = OpenStreetMap.org (OSM) ni mradi wa jumuiya wa kujenga ramani isiyolipishwa na iliyo wazi. Ndiyo chanzo kikuu cha data ya ramani katika Ramani za Kikaboni na inafanya kazi sawa na Wikipedia. Unaweza kuongeza au kubadilisha maeneo na yanapatikana kwa mamilioni ya watumiaji kote Ulimwenguni.\nJiunge na jumuiya na usaidie kutengeneza ramani bora kwa kila mtu!
    th = OpenStreetMap.org (OSM) เป็นโครงการชุมชนเพื่อสร้างแผนที่แบบเปิดและเสรี เป็นแหล่งข้อมูลแผนที่หลักในแผนที่ทั่วไปและทำงานคล้ายกับวิกิพีเดีย คุณสามารถเพิ่มหรือแก้ไขสถานที่ได้ และสถานที่เหล่านั้นจะพร้อมให้บริการแก่ผู้ใช้หลายล้านคนทั่วโลก\nเข้าร่วมชุมชนและช่วยสร้างแผนที่ที่ดีขึ้นสำหรับทุกคน!
    tr = OpenStreetMap.org (OSM) ücretsiz ve açık bir harita oluşturmak için bir topluluk projesidir. Organik Haritalar'daki harita verilerinin ana kaynağıdır ve Wikipedia'ya benzer şekilde çalışır. Yerleri ekleyebilir veya düzenleyebilirsiniz ve bunlar tüm dünyadaki milyonlarca kullanıcı tarafından kullanılabilir hale gelir.\nTopluluğa katılın ve herkes için daha iyi bir harita oluşturmaya yardımcı olun!

>     uk = OpenStreetMap.org (OSM) - це громадський проект зі створення вільної та відкритої карти. Це основне джерело картографічних даних в Organic Maps і працює подібно до Вікіпедії. Ви можете додавати або редагувати місця, і вони стають доступними мільйонам користувачів по всьому світу.\nПриєднуйтесь до спільноти і допоможіть зробити карту кращою для всіх!

Reviewer: ```suggestion
    uk = OpenStreetMap.org (OSM) - це громадський проект зі створення вільної та відкритої мапи. Це основне джерело картографічних даних в Organic Maps і працює подібно до Вікіпедії. Ви можете додавати або редагувати місця, і вони стають доступними мільйонам користувачів по всьому світу.\nПриєднуйтесь до спільноти і допоможіть зробити мапу кращою для всіх!
```

    vi = OpenStreetMap.org (OSM) là một dự án cộng đồng nhằm xây dựng bản đồ mở và miễn phí. Đây là nguồn dữ liệu bản đồ chính trong Bản đồ không phải trả tiền và hoạt động tương tự như Wikipedia. Bạn có thể thêm hoặc chỉnh sửa địa điểm và chúng sẽ có sẵn cho hàng triệu người dùng trên toàn thế giới.\nTham gia cộng đồng và giúp tạo bản đồ tốt hơn cho mọi người!
    zh-Hans = OpenStreetMap.org（OSM）是一个构建免费开放地图的社区项目。它是有机地图中地图数据的主要来源，其工作原理类似于维基百科。你可以添加或编辑地点，全世界数以百万计的用户都可以使用它们！
    zh-Hant = OpenStreetMap.org (OSM) 是一個建立免費開放地圖的社群專案。它是有機地圖中地圖數據的主要來源，其工作方式類似於維基百科。您可以新增或編輯地點，全世界數百萬用戶都可以使用它們。

  [login_to_make_edits_visible]
    tags = android,ios
    en = Create an OpenStreetMap account or log in to publish your map edits to the world.
    af = Skep 'n OpenStreetMap-rekening of meld aan om jou kaartwysigings aan die wêreld te publiseer.
    ar = أنشئ حساباً على OpenStreetMap أو سجّل الدخول لنشر تعديلاتك على الخريطة للعالم.
    az = OpenStreetMap hesabı yaradın və ya xəritə redaktələrinizi dünyada dərc etmək üçün daxil olun.

>     be = Стварыце ўліковы запіс OpenStreetMap або ўвайдзіце ў сістэму, каб апублікаваць праўкі карты ўсім свеце.

Reviewer: ```suggestion
    be = Стварыце ўліковы запіс OpenStreetMap або ўвайдзіце ў сістэму, каб апублікаваць праўкі мапы ўсім на свеце.
```

    zh-Hans = 截至 %@ 的社区创建的 OpenStreetMap 数据。请访问 OpenStreetMap.org 以了解有关如何编辑和更新地图的信息。
    zh-Hant = 截至 %@ 的社群創建的 OpenStreetMap 資料。請訪問 OpenStreetMap.org 以了解有關如何編輯和更新地圖的更多資訊。

>   [login_osm_presentation]

Reviewer 2: so above `[osm_presentation]` is not used in Android anymore - please remove the tag

Author: It's till used in the About OSM screen
